apache/zeppelin
# [Zeppelin-2572] multiple paragraphs actions

### What is this PR for?
This **PR** adds the ability to perform actions on several paragraphs simultaneously. He adds the checkbox to each paragraph. When you select at least one paragraph in the action bar, buttons appear that perform actions on the selected paragraphs.


### What type of PR is it?
Feature

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-2572

### How should this be tested?
**1.** Select several paragraphs (In the actinBar appear buttons with actions for these paragraphs)
**2.** Perform several actions on paragraphs (by pressing the buttons)
**3.** Click "Reset selection" button

### Screenshot

![gif](https://user-images.githubusercontent.com/25951039/33115591-993e6cf4-cf72-11e7-8987-64cb3e137ba8.gif)

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no
